.github/workflows/deploy.yaml

```diff
@@ -51,8 +51,6 @@ jobs:
       - name: '[BUILD] Install dependencies / link / build'
         run: |
           yarn
-          yarn setup
-          yarn util:link
```

Comment:
must have been legacy code that has been used with lerna

Reply:
correct